### PR TITLE
Added more details in changelog for icon clean up and fixed issue in link

### PR DESCRIPTION
**Pull request by dafortin**
**This project uses [semver](http://semver.org), please check the scope of this pr:**

- [x] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

# CHANGELOG
* **Added** more details in changelog for icon clean up
* **Fixed** link when route/routeName is not set and that we are using `text` attribute

### CODE REVIEW

**Comment by sglanzer-fire** [comment]
Approved - Thanks @dafortin - nice to have the full set of details

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-core/pull-request/347/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-core)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,120 +1,157 @@
 # 1.3.0
-
-* Updated frost-link to accept named parameters in addition to position parameters, which unblocks the `spread` use case
-
+
+* Updated frost-link to accept named parameters in addition to position parameters, which unblocks the `spread` use case
+
```
Comment (job13er) [inline CHANGELOG.md:10]: @dafortin Are you using Windows by any chance? Or an editor that uses carriage-returns? I'm noticing what appear to be line ending changes here.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,120 +1,157 @@
...
+- Icon - application => **Moved** to ember-frost-navigation
+- Icon - arrow => **Deleted**
+- Icon - back => **Deleted**
+- Icon - chevron-left => **Renamed** chevron
+- Icon - close-circle => **Renamed** round-close
+- Icon - dialog-error => **Replaced** by error
+- Icon - double-chevron-left => **Renamed** chevron-double
+- Icon - hide => **Deleted**
+- Icon - home => **Deleted**
+- Icon - info => **Moved** to ember-frost-modal & ember-frost-notifier & ember-frost-fields
```
Comment (job13er) [inline CHANGELOG.md:126]: Just to make sure I understand this, we're moving an icon from core to live in three different addons? So, when UX changes that icon, we're gonna have to change three copies of it?

File: CHANGELOG.md (hunk 1 above)
Comment (dafortin) [reply CHANGELOG.md:10]: No I'm not.

File: CHANGELOG.md (hunk 2 above)
Comment (dafortin) [reply CHANGELOG.md:126]: That's what was requested by @sglanzer.

**Pull request merged**